### PR TITLE
docs(README): minor corrections (AEROGEAR-9256)

### PR DESCRIPTION
**Pull request by damienomurchu**
## Motivation
fix minor issues in README

## What
fix minor issues in README

## Why
fix minor issues in README

## How
fix minor issues in README

## Verification Steps

Check README & verify changes are ok.

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -226,8 +226,8 @@ NOTE: The operator, application and database will be installed in the namespace
 +
 |===
 | *Resource*    | *Description*
-| link:./pkg/controller/mobilesecurityservice/configmaps.go[configmaps.go]             | Define the ConfigMap resources required for the Mobile Security Service Application and its Database. It will create the mobile-security-service-app` which map the values used in the Environment Variables of both.
-| link:./pkg/controller/mobilesecurityservice/deployments.go[deployments.go]           | Define the Deployment resource of Mobile Security Service Application. (E.g container and resources definitions)
+| link:./pkg/controller/mobilesecurityservice/configmaps.go[configmaps.go]             | Define the ConfigMap resources required for the Mobile Security Service Application and its Database. It will create the `mobile-security-service-app` which map the values used in the Environment Variables of both.
+| link:./pkg/controller/mobilesecurityservice/deployments.go[deployments.go]           | Define the Deployment resource of Mobile Security Service Application, ie container and resources definitions.
```
Comment (camilamacedo86) [inline README.adoc]: ie ?

Comment (damienomurchu) [reply README.adoc]: same as `eg`

Comment (damienomurchu) [reply README.adoc]: not quite the same actually (https://www.thoughtco.com/when-to-use-common-latin-abbreviations-116688) - have switched back to `e.g.` :)

File: README.adoc [hunk 2]
```diff
@@ -86,7 +86,7 @@ To verify the installation has been successful run the following
 [source,shell]
 ----
 # check the project exists 
-$ oc project mobile-security-service-operator
+$ oc project mobile-security-service
```
Comment (laurafitzgerald) [inline README.adoc]: There's a variable you can use for this: https://github.com/aerogear/mobile-security-service-operator/pull/108/files#diff-7bd4a925c695c2eb0eced3872b9b965fR9

Comment (damienomurchu) [reply README.adoc]: not sure this can be used when the adoc file is not being built, so will just go with hardcoded value here

Comment (laurafitzgerald) [reply README.adoc]: We use the variable elsewhere in this adoc file

**Pull request merged**